### PR TITLE
[zh] Sync #14788 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #14788 into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/ops/ambient/getting-started/index.md [hunk 1]
```diff
@@ -252,7 +256,18 @@ $ export GATEWAY_SERVICE_ACCOUNT=ns/istio-system/sa/bookinfo-gateway-istio
 
 ## 添加应用到 Ambient 网格 {#addtoambient}
 
-您只需给命名空间打标签，就可以作为 Ambient 网格的一部分，在给定的命名空间启用所有 Pod：
+当应用程序 Pod 是 Ambient 网格的一部分时，
+您可以检查 ztunnel 代理日志以确认网格正在对流量进行重定向。
+在我们将命名空间标记为 Ambient 网格的一部分之前，
+请检查与 `inpod` 相关的 ztunnel 日志，该日志指示已启用 in-Pod 重定向模式：
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/getting-started/index.md:262]:
```suggestion
请检查与 `inpod` 相关的 ztunnel 日志，确认 in-Pod 重定向模式已被启用：
```

File: content/zh/docs/ops/ambient/getting-started/index.md [hunk 2]
```diff
@@ -252,7 +256,18 @@ $ export GATEWAY_SERVICE_ACCOUNT=ns/istio-system/sa/bookinfo-gateway-istio
...
+
+{{< text bash >}}
+$ kubectl logs ds/ztunnel -n istio-system  | grep inpod_enabled
+inpod_enabled: true
+{{< /text >}}
+
+现在，您可以通过简单地标记命名空间来使给定命名空间中的所有
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/getting-started/index.md:269]:
```suggestion
现在，您只需标记命名空间就能使给定命名空间中的所有
```

File: content/zh/docs/ops/ambient/getting-started/index.md [hunk 3]
```diff
@@ -261,6 +276,14 @@ $ kubectl label namespace default istio.io/dataplane-mode=ambient
 恭喜！您已成功将 default 命名空间中的所有 Pod 添加到网格中。
 请注意，您不必重新启动或重新部署任何内容！
 
+再次检查代理的 ztunnel 日志是否已收到有关 Ambient
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/getting-started/index.md:279]:
```suggestion
再次检查 ztunnel 日志，确认代理已收到有关 Ambient
```

File: content/zh/docs/ops/ambient/getting-started/index.md [hunk 4]
```diff
@@ -261,6 +276,14 @@ $ kubectl label namespace default istio.io/dataplane-mode=ambient
...
+应用程序 Pod 的网络命名空间（netns）信息，并已开始为其代理：
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/getting-started/index.md:280]:
```suggestion
应用程序 Pod 的网络命名空间（netns）信息，并已开始为此命名空间执行代理：
```

**Review by windsonsea — COMMENTED** [review]
some text suggestions

File: content/zh/docs/ops/ambient/getting-started/index.md [hunk 5]
```diff
@@ -287,7 +310,8 @@ $ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/ | grep -o "<t
 
 ## 确保应用访问安全 {#secure}
 
-将您的应用添加到 Ambient 模式网格之后，可以使用 Layer 4 鉴权策略确保应用访问的安全。
+将您的应用添加到 Ambient 模式网格之后，
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/getting-started/index.md:313]:
```suggestion
将您的应用添加到 Ambient 模式之后，
```
这个是不是 Ambient 模式就可以了？

**Review by windsonsea — APPROVED** [review]
lgtm

**Pull request merged**